brvs-core/src/main/java/iroha/validation/listener/IrohaReliableChainListener.java

```diff
@@ -132,6 +121,7 @@ public IrohaReliableChainListener(
     logger.info("Subscribed to Iroha chain listener");
 
     return source.map(delivery -> {
+          Thread.sleep(IROHA_WORLD_STATE_UPDATE_TIME);
```

Comment:
Why we do this?

Reply:
>Added kostyl of delaying block consumption because sometimes Iroha world state view is not updated fast enough after block publishing (very strange, reported to Iroha team)